components/asset.rst

```diff
@@ -72,7 +72,7 @@ any versioning::
     echo $package->getUrl('image.png');
     // result: image.png
 
-Packages implement :class:`Symfony\\Component\\Asset\\PackageInterface`,
+Package implement :class:`Symfony\\Component\\Asset\\PackageInterface`,
```

Comment:
If we change `packages` -> `package` then we should also change `implement` -> `implements`

In any case, I think that the original text is correct because it says that all packages implement a certain interface. Maybe I'm missing something? Thanks!

Reply:
Yes. I agree.
Thanks.